spec/model/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
+
+  before { subject.save }
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'posts_counter should be an integer' do
+    subject.posts_counter = 'invalid'
+    expect(subject).to_not be_valid
+  end
+end
```

Comment:
- [ ] It is a requirement that all custom methods in the models should be tested, so please do so as to meet the requirements.
The methods to test from the user model are:

- `recent_posts`
- `update_posts_counter`

## Again here note: (Optional)
You do not need the `update_posts_counter` in the user model, you could remove it and just test the `recent_posts` method. The `update_posts_counter` method should be put in the post model and tested from there